src/vs/workbench/contrib/terminal/browser/links/terminalWordLinkDetector.ts

```diff
@@ -68,6 +70,22 @@ export class TerminalWordLinkDetector implements ITerminalLinkDetector {
 				},
 				startLine
 			);
+
+			// vscode:// protocol links
+			if (word.text.startsWith(`${Schemas.vscode}://`)) {
```

Comment:
I think this would only detect protocol URLs for VS Code stable even when in VS Code Insiders. For VS Code insiders the protocol URL scheme is `vscode-insiders://`. This is generically available as [`IProductService.urlProtocol`](https://github.com/microsoft/vscode/blob/0e20196eae2e98186c881b00de5eb6a9715264bf/src/vs/base/common/product.ts#L48), but maybe we want to detect both `vscode://` and `vscode-insiders://` links regardless of quality?

Reply:
For OSS the url protocol scheme is [`code-oss`](https://github.com/microsoft/vscode/blob/0e20196eae2e98186c881b00de5eb6a9715264bf/product.json#L30) so maybe we should just defer to what's defined in the product service.

Reply:
Tweaked to make this work. I tried to more generally support URIs but struggled so will stick to vscode ones only for now.